src/v3/plugins/github/nodes.js

```diff
@@ -47,6 +47,48 @@ export type StructuredAddress =
   | CommentAddress
   | UserlikeAddress;
 
+// Each of these types has 0 or more "AUTHORS" edges,
```

Comment:
From Discord:

> When a GitHub account is deleted, the post stays, but it has a null
> author as far as the graphql api is concerned

This makes sense, and seems like exactly the kind of thing that should
go in this docstring. Otherwise, arrogant programmers like me will
assume that of course an `AuthorableAddress` always has at least one
author, and will write broken code. :-)